magma/lib/magma/attributes/file.rb

```diff
@@ -68,10 +47,100 @@ def query_to_tsv(file)
       file ? file[:url] : nil
     end
 
-    def entry(value, loader)
+    def entry(file, loader)
+      value = case file[:path]
+      when '::blank'
```

Comment:
Does this work?  There's a return below so these expressions don't do anything I believe.

Reply:
This seems to work, when I check in the database, for `::blank`, `::temp`, and `null`. `value` gets set to the right hash value depending on the case (and does not update the database for `::temp` but does return an upload URL).

Reply:
OHHHHH now I'm reading this right.  The value is being set to the expression of the case.